pkg/network/protocols/kafka/kafka_statkeeper.go

```diff
@@ -0,0 +1,69 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+//go:build linux_bpf
+// +build linux_bpf
+
+package kafka
+
+import (
+	"github.com/DataDog/datadog-agent/pkg/network/config"
+	"sync"
+)
+
+type KafkaStatKeeper struct {
+	stats      map[Key]*RequestStat
+	statsMutex sync.RWMutex
+	maxEntries int
+	telemetry  *Telemetry
+}
+
+func NewKafkaStatkeeper(c *config.Config, telemetry *Telemetry) *KafkaStatKeeper {
+	return &KafkaStatKeeper{
+		stats:      make(map[Key]*RequestStat),
+		maxEntries: c.MaxKafkaStatsBuffered,
+		telemetry:  telemetry,
+	}
+}
+
+func (statKeeper *KafkaStatKeeper) Process(tx *EbpfKafkaTx) {
+	statKeeper.add(tx)
+}
+
+func (statKeeper *KafkaStatKeeper) add(transaction *EbpfKafkaTx) {
+	key := Key{
+		RequestAPIKey:  transaction.Request_api_key,
+		RequestVersion: transaction.Request_api_version,
```

Comment:
you implemented a wrapper function for them (at least for one of them), let's be consistent and use the wrappers